source/testing/acceptance.md

```diff
@@ -114,21 +146,26 @@ Synchronous helpers are performed immediately when triggered.
 
 ### Wait Helpers
 
+Instead of using `async` and `await` to wait for asynchronous helpers to complete,
+you may also use these test helpers which will do the waiting for you.
 The `andThen` helper will wait for all preceding asynchronous helpers to
 complete prior to progressing forward. Let's take a look at the following
 example.
 
 ```tests/acceptance/new-post-appears-first-test.js
-import { test } from 'qunit';
-import moduleForAcceptance from 'my-app/tests/helpers/module-for-acceptance';
-
-moduleForAcceptance('Acceptance | posts');
-
-test('should add new post', function(assert) {
-  visit('/posts/new');
-  fillIn('input.title', 'My new post');
-  click('button.submit');
-  andThen(() => assert.equal(find('ul.posts li:first').text(), 'My new post'));
+import { module, test } from 'qunit';
+import { click, fillIn, visit } from '@ember/test-helpers';
+import { setupApplicationTest } from 'ember-qunit';
+
+module('Acceptance | posts', function(hooks) {
+  setupApplicationTest(hooks);
+
+  test('should add new post', function(assert) {
+    visit('/posts/new');
+    fillIn('input.title', 'My new post');
+    click('button.submit');
+    andThen(() => assert.equal(this.element.querySelector('ul.posts li:first').textContent, 'My new post'));
```

Comment:
`andThen` is completely gone in tests that are setup with `setupApplicationTest` (it is only available in "legacy" acceptance tests)

Reply:
Sweet, I wasn't sure about that yet. I can update / remove that section then to only discuss the `async / await` syntax instead